modules/admission-controller-default.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * architecture/admission-controllers.adoc
+
+[id="admission-controller-default_{context}"]
+= Default admission controllers
+
+//Future xref - A set of default admission controllers are enabled in {product-title} {product-version}. These default admission controllers contribute to fundamental control plane functionality, such as ingress admission policy, xref:../nodes/clusters/nodes-cluster-overcommit.adoc#nodes-cluster-resource-override_nodes-cluster-overcommit[cluster resource limit override] and quota policy.
```

Comment:
avoid too many terms for the same thing: plugin, controller, policy

Reply:
Resolved as per our conversation today.